vmdb/app/models/widget_import_validator.rb

```diff
@@ -1,8 +1,11 @@
 class WidgetImportValidator
+  class NonValidWidgetYamlError < StandardError; end
```

Comment:
Either way, I guess. I think I just kinda saw the previous "NonYamlError" and went with NonValid. I guess maybe NonValid implies that the YAML is OK but that it's not actually widget YAML, whereas Invalid implies that it's not YAML at all? I could go either way.